script.js

```diff
@@ -1,4 +1,68 @@
 var main = function (input) {
   var myOutputValue = 'hello world';
-  return myOutputValue;
-};
+  var words = computerGenResult();
+  if(input == 'scissors' && words == 2){
```

Comment:
This is bad representation that you are hard-coding the numbers 1, 2, 3 as a proxy for scissors, paper or stone. It's not clear to a reader what word 1, 2, or 3 represent. Better representation would have `computerGenResult` generate a string, one of 'scissors', 'paper', or 'stone', and compare the string to the user input.